user_docs/en/userGuide.t2t

```diff
@@ -267,6 +267,64 @@ To open NVDA's menu, press ``NVDA+n``.
 To open NVDA's general settings dialog directly, press ``NVDA+control+g``.
 Many settings screens have keystrokes to open them directly, such as ``NVDA+control+s`` for synthesizer, or ``NVDA+control+v`` for other voice options.
 
+++ Add-ons ++[Addons]
+Add-ons are like small programs which provide new or changed functionality for NVDA.
+Add-ons are developed by the NVDA community, or external companies.
+As with any software, it is important to be confident in the developer of an add-on before using it.
+The "Add-ons and the Add-on Store" section of the user guide contains information about every feature of the add-on store.
+
+The add-on store is available from the Tools menu.
+Press ``NVDA+n`` to open the NVDA menu, then ``t`` for tools, then ``a`` for add-on store.
+When the add-on store opens, it shows "Available add-ons" if no add-ons are installed.
+When add-ons are installed, the focus starts in the "Installed add-ons" tab when the add-on store opens.
+
++++ Available add-ons +++[AvailableAddons]
+When the window first opens, add-ons may take a few seconds to load.
+NVDA will read the name of the first add-on once the list of add-ons finishes loading.
+Available add-ons are listed alphabetically in a grid.
+Use the ``arrow keys`` or press the first letter of an add-on name to move around the list.
+Press ``tab`` once to move to a description of the currently selected add-on.
+Use the reading or ``arrow keys`` to read the full description.

```

Comment:
I'm not familiar with the term reading here, is it something we use in the rest of the docs?